src/components/app/flyouts/settings/platformSettings.js

```diff
@@ -0,0 +1,194 @@
+      this.setState({
+        stillInitializing: false
+      });
+    }
+  }
+
+  renderSvgLogo = (logo) => {
+    return (
+      <Svg path={logo} className="logo-svg" />
+    );
+  };
+
+  renderUploadContainer = () => {
+    const fileName = this.state.newLogoName
+    const { t } = this.props;
+    const { logoIsDefault, isValidFile } = this.state;
+    const fileNameClass = isValidFile ? 'file-name-valid' : 'file-name-invalid';
+    return (
+      <div className="upload-logo-name-container">
+        <div className="upload-logo-container">
+          <div className="image-preview">
+            {logoIsDefault ?
+              this.renderSvgLogo(this.state.currentLogo)
+              :
+              <img className="logo-img" src={this.state.previewLogo} alt={t('platformSettings.previewLogo')} />
+            }
+          </div>
+          <div className="replace-logo">{t('platformSettings.replaceLogo')}</div>
+          <div className="upload-btn-container">
+            <FileInput className="upload-button" classForLabel="description" isEdit={true} onChange={this.onUpload}
+              accept=".jpg, .jpeg, .png, .svg" label={t('platformSettings.upload')} t={t} />
+            <div className="file-upload-feedback">
+              {isValidFile ?
+                <Svg className="checkmark" path={svgs.checkmark} alt={t('platformSettings.checkmark')} />
+                :
+                fileName && <Svg className="invalid-file-x" path={svgs.x} alt={t('platformSettings.error')} />
+              }
+            </div>
+            <div className={fileNameClass}>{fileName}</div>
+          </div>
+          {!isValidFile && fileName &&
+            <div className="upload-error-message">
+              <Svg className="upload-error-asterisk" path={svgs.error} alt={t('platformSettings.error')} />
+              {t('platformSettings.uploadError')}
+            </div>
+          }
+          <Section.Content className="platform-section-description show-line-breaks">{t('platformSettings.logoDescription')}</Section.Content>
+        </div>
+        <Section.Content className="name-input-container">
+          <div className="section-subtitle">{t('platformSettings.applicationName')}</div>
+          <FormControl type="text" className="name-input long" classForLabel="description"
+            isEdit={true} placeholder={this.state.currentApplicationName} onChange={this.onNameChange} />
+        </Section.Content>
+      </div>
+    );
+  }
+
+  render() {
+    const { t } = this.props;
+    const { logoIsDefault, validating } = this.state;
+    return (
+      <Section.Container collapsable={true} className="setting-section">
+        <Section.Header>{t('platformSettings.nameAndLogo')}</Section.Header>
+        <Section.Content>{t('platformSettings.nameLogoDescription')}</Section.Content>
+        {this.state.stillInitializing ?
+          <Indicator size='medium' />
+          :
+          <Section.Content>
+            {this.state.edit ?
```

Comment:
Is this the formatting we are following 
`{this.state.edit ?`
or should it be
```
{
    this.state.edit ?
    ...
}
```

Reply:
it looks like format from old code base was:
{condition
   ? first
   : second}